src/tinohelm/data/catalog.py

```
@@ -1384,10 +1461,42 @@ def _remote_catalog_constructor_args(catalog_root: Path, storage: Any) -> tuple[
+    works like MarkPriceUpdate/FundingRateUpdate.
+    """
+    global _NT_UPDATE_QUERY_SUPPORT_READY
+
+    if _NT_UPDATE_QUERY_SUPPORT_READY:
+        return
+
+    try:
+        from nautilus_trader.model.data import IndexPriceUpdate
+        from nautilus_trader.serialization.arrow.serializer import make_dict_deserializer, register_arrow
+        from nautilus_trader.serialization.arrow.schema import NAUTILUS_ARROW_SCHEMA
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)